index.c

```diff
@@ -1852,12 +1852,17 @@ int mutt_index_menu(struct MuttWindow *dlg)
         menu->redraw = REDRAW_FULL;
         break;
 
-      case OP_SEARCH:
+      // Initiating a search can happen on an empty mailbox, but
+      // searching for next/previous/... needs to be on a message and
+      // thus a non-empty mailbox
       case OP_SEARCH_REVERSE:
       case OP_SEARCH_NEXT:
       case OP_SEARCH_OPPOSITE:
         if (!prereq(Context, menu, CHECK_IN_MAILBOX | CHECK_MSGCOUNT | CHECK_VISIBLE))
           break;
```

Comment:
Please add a `/* fallthrough */` comment here to show that the behaviour is expected.

Reply:
Added :+1: